packages/frontend/src/components/common/Table/ScrollableTable/TableBody.tsx

```diff
@@ -223,7 +231,14 @@ const VirtualizedTableBody: FC<{
     const rowVirtualizer = useVirtualizer({
         getScrollElement: () => tableContainerRef.current,
         count: rows.length,
-        estimateSize: () => ROW_HEIGHT_PX,
+        estimateSize: (index) => {
+            try {
+                return getMaxRowHeight(rows[index]);
+            } catch (e) {
+                console.error('Error getting row height', e);
```

Comment:
I don't expect this method to fail, but just in case, I added a try-catch so we don't throw a runtime error if this fails